README.md

```diff
@@ -157,6 +157,18 @@ Add the configurations to the `custom.yml`. A sample is given below.
 
 Follow the steps mentioned under `docs` directory to customize/create new Ansible scripts and deploy the recommended patterns.
 
+### 3. Including custom Keystore and Truststore
```

Comment:
Shall we add this under customisations? WDYT?

Reply:
@Savidude : let us know your input on this

Reply:
Changed the content as needed